e2e/generateReferences.ts

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const puppeteer = require('puppeteer')
+const fs = require('fs');
```

Comment:
Why do we use `require` here instead of `import`?

Reply:
It's ran via `node-ts` which does not work quite well with ES6 from my experience.